test/node/connection-inspect.js

```diff
@@ -0,0 +1,97 @@
+'use strict';
+
+const test = require('tap');
+
+const jstp = require('../..');
+
+const app = require('../fixtures/application.js');
+const application = new jstp.Application(app.name, app.interfaces);
+
+let server;
+let client;
+
+test.beforeEach((done) => {
+  server = jstp.tcp.createServer(0, [application]);
+  server.listen(() => {
+    const port = server.address().port;
+    client = jstp.tcp.createClient({ host: 'localhost', port });
+    done();
+  });
+});
+
+test.afterEach((done) => {
+  client.disconnect();
+  server.close();
+  done();
+});
+
+test.test('must process inspect packets', (test) => {
+  const expectedInterfaces = Object.keys(app.interfaces);
+  let expectedTests = 1;
+  expectedInterfaces.forEach((iface) => {
+    expectedTests += 1;
+    expectedTests += Object.keys(app.interfaces[iface]).length;
+  });
+
+  test.plan(expectedTests);
+  client.connectAndHandshake(app.name, null, null,
+    (error, connection) => {
+      test.assertNot(error, 'must connect to server');
+
+      expectedInterfaces.forEach((iface) => {
+        connection.inspectInterface(iface, (error, methods) => {
+          test.assertNot(error, `must inspect ${iface}`);
+          Object.keys(app.interfaces[iface]).forEach((method) => {
```

Comment:
You may avoid using braces and parenthesis here, there is just one statement in this function.